### ISSUE TITLE
[BUG]: ##[warning]Resource file has already set to: /agent/_work/_tasks/Kubernetes_cbc316a2-586f-4def-be79-488a1f503564/1.241.5/node_modules/azure-pipelines-tasks-azure-arm-rest/module.json

### ISSUE THREAD

**Issue by iondrugalea** [microsoft/azure-pipelines-tasks#20224]
### New issue checklist

- [X] I searched for [existing GitHub issues](https://github.com/microsoft/azure-pipelines-tasks/issues)
- [X] I read [pipeline troubleshooting guide](https://docs.microsoft.com/vsts/build-release/actions/troubleshooting)
- [X] I checked [how to collect logs](https://learn.microsoft.com/azure/devops/pipelines/troubleshooting/review-logs?view=azure-devops)

### Task name

Kubernetes@1

### Task version

1

### Issue Description

A bug happened!
I use in my azure devops pipline the task Kubernetes@1 for diferent use cases. Since about 1 Month i see this warning everytime when the Kubernetes@1 is executed. I call the Kubernetes@1 multiple times and each time i see this warning. 

### Environment type (Please select at least one enviroment where you face this issue)

- [X] Self-Hosted
- [ ] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

Ubuntu 22.04

### Relevant log output

```shell
##[warning]Resource file has already set to: /agent/_work/_tasks/Kubernetes_cbc316a2-586f-4def-be79-488a1f503564/1.241.5/node_modules/azure-pipelines-tasks-azure-arm-rest/module.json
```

### Full task logs with system.debug enabled

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### Repro steps

```yml
- task: Kubernetes@1
    displayName: 
    inputs:
      connectionType: "Kubernetes Service Connection"
      kubernetesServiceEndpoint: " ${{ parameters.kubernetesServiceEndpoint }}"
      command: "apply"
      useConfigurationFile: true
      configuration: 
      arguments: 
      namespace:
```

**Comment by andylamb-rlg** [comment]
We're also seeing this issue in the [DotNetCoreCLI@2](https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli) Pipeline Task, version 2.244.1

**Comment by cabralRodrigo** [comment]
![image](https://github.com/user-attachments/assets/52795713-1da9-494f-a0a1-c87281ee9b5d)
Me too.

**Comment by stuartag** [comment]
Same here for DotNetCoreCLI@2.245.1 but was also on previous versions for the past month maybe?

**Comment by bittebak** [comment]
Same here. 
Azure hosted
`Task         : .NET Core
Description  : Build, test, package, or publish a dotnet application, or run a custom dotnet command
Version      : 2.245.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli
==============================================================================
##[warning]Resource file has already set to: D:\a\_tasks\DotNetCoreCLI_5541a522-603c-47ad-91fc-a4b1d163081b\2.245.1\node_modules\azure-pipelines-tasks-packaging-common\module.json
`
I manage multiple DevOps tenants. All pipelines on multiple tenants have the same issue.

It still works though.

**Comment by jackhorton** [comment]
Looks like the DotNetCoreCLI@2 issue should be fixed in #20427

**Comment by EshitaD** [comment]
Closing the issue, PR has been merged for this issue.
PR - https://github.com/microsoft/azure-pipelines-tasks/pull/20308
